app/models/registration.ts

```diff
@@ -11,13 +11,13 @@ import RegistryProviderModel from './registry-provider';
 import UserModel from './user';
 
 export enum RegistrationState {
-    Embargoed = 'embargoed',
-    Public = 'public',
-    Withdrawn = 'withdrawn',
-    PendingRegistration = 'pendingRegistrationApproval',
-    PendingWithdrawal = 'pendingWithdrawal',
-    PendingEmbargo = 'pendingEmbargoApproval',
-    PendingEmbargoTermination = 'pendingEmbargoTerminationApproval',
+    embargoed = 'embargoed',
+    public = 'public',
+    withdrawn = 'withdrawn',
+    pendingRegistrationApproval = 'pendingRegistrationApproval',
+    pendingWithdrawal = 'pendingWithdrawal',
+    pendingEmbargoApproval = 'pendingEmbargoApproval',
+    pendingEmbargoTerminationApproval = 'pendingEmbargoTerminationApproval',
```

Comment:
Drive-by comment: We've been following [this styleguide's recommendations](https://basarat.gitbooks.io/typescript/docs/styleguide/styleguide.html#enum) for enum casing (`PascalCase` for enum names *and* members). This is also the case used by the [official TypeScript handbook](https://www.typescriptlang.org/docs/handbook/enums.html). I like it because it makes it clear in usage that it's  an enum and not an object (which would use `camelCase` for members).

Reply:
what @jamescdavis said

Reply:
There's not much we can do about this - I'd like to remove this enum completely but that would have to be in a separate PR. We're not really using this enum in an enum capacity.

Reply:
I'll see what I can do but we might have to divorce the idea of this registration state from the state function below.

Reply:
I agree an enum doesn't really serve us well here. I see why you changed the case.

FWIW, if we could guarantee order with enums you *could* do:
```ts
get state(): RegistrationState {
    for (const state of (Object.keys(RegistrationState) as Array<keyof typeof RegistrationState>)) {
        if (this[RegistrationState[state]]) {
            return RegistrationState[state];
        }
    }
    return RegistrationState.Public;
}
```
but I don't think we can because enums are transpiled to objects.

Reply:
We basically are doing that in the registrationstate, and the enum is integral to other bits. It was pretty easy to change to meet this specification, however.